app/src/main/java/io/pslab/activity/DataLoggerActivity.java

```diff
@@ -62,6 +62,10 @@ protected void onCreate(Bundle savedInstanceState) {
                 getSupportActionBar().setTitle(caller);
                 categoryData = LocalDataLog.with().getTypeOfSensorBlocks(getString(R.string.multimeter));
                 break;
+            case "Compass":
+                getSupportActionBar().setTitle(caller);
```

Comment:
What do you think @neel1998 , since this line is the same for all these cases, would it throw any error or malfunction if we put `getSupportActionBar().setTitle(caller);` before the switch statement?

Reply:
I think that can be done. I am on it

Reply:
I mean, first check if it's breaking anything. If not you can update it here itself :+1:

Reply:
The changes are working fine